docs/02_cleos/03_command-reference/get/kv_table.md

```diff
@@ -12,48 +12,393 @@ Retrieves the contents of a database kv_table
+```sh
+cleos get kv_table --encode-type dec -i 1 contr_acct kvtable foo
+```
+```json
+{
+    "rows": [
+        "000000000000600e3d010000000000000004626f6261"
+    ],
+    "more": false,
+    "next_key": ""
+}
+```
+
+Range query to return all rows that match eosio name keys between `bobd` (inclusive) and `bobh` (exclusive) from kv_table named `kvtable` owned by `contr_acct` account using kv_table index `primarykey`:
+```sh
+cleos get kv_table --encode-type name -L bobd -U bobh contr_acct kvtable primarykey -b
+```
+```json
+{
+    "rows": [
+        "000000000000900e3d040000000000000004626f6264000000",
+        "000000000000a00e3d050000000000000004626f6265000000",
+        "000000000000b00e3d060000000000000004626f6266000000",
+        "000000000000c00e3d070000000000000004626f6267000000"
+    ],
+    "more": false,
+    "next_key": ""
+}
+```
+
+Range query to return all rows (in reverse order) that match eosio name keys between `bobd` (exclusive) and `bobh` (inclusive) from kv_table named `kvtable` owned by `contr_acct` account using kv_table index `primarykey`:
+```sh
+cleos get kv_table --encode-type name -L bobd -U bobh contr_acct kvtable primarykey -b -r
+```
+```json
+{
+    "rows": [
+        "000000000000d00e3d080000000000000004626f6268000000",
+        "000000000000c00e3d070000000000000004626f6267000000",
+        "000000000000b00e3d060000000000000004626f6266000000",
+        "000000000000a00e3d050000000000000004626f6265000000"
+    ],
+    "more": false,
+    "next_key": ""
+}
+```
+
+Range query to return all rows starting from eosio name key `bobg` (inclusive) from kv_table named `kvtable` owned by `contr_acct` account using kv_table index `primarykey`:
+```sh
+cleos get kv_table --encode-type name -L bobg contr_acct kvtable primarykey -b
+```
+```json
+{
+    "rows": [
+        "000000000000c00e3d070000000000000004626f6267000000",
+        "000000000000d00e3d080000000000000004626f6268000000",
+        "000000000000e00e3d090000000000000004626f6269000000",
+        "000000000000f00e3d0a0000000000000004626f626a000000"
+    ],
+    "more": false,
+    "next_key": ""
+}
+```
+
+Range query to return all rows (in reverse order) starting from eosio name key `bobg` (exclusive) from kv_table named `kvtable` owned by `contr_acct` account using kv_table index `primarykey`:
+```sh
+cleos get kv_table --encode-type name -L bobg contr_acct kvtable primarykey -b -r
+```
+```json
+{
+    "rows": [
+        "000000000000f00e3d0a0000000000000004626f626a000000",
+        "000000000000e00e3d090000000000000004626f6269000000",
+        "000000000000d00e3d080000000000000004626f6268000000"
+    ],
+    "more": false,
+    "next_key": ""
+}
+```
+
+Range query to return all rows until eosio name key `bobe` (exclusive) from kv_table named `kvtable` owned by `contr_acct` account using kv_table index `primarykey`:
+```sh
+cleos get kv_table --encode-type name -U bobe contr_acct kvtable primarykey -b
```

Comment:
missing -L boba

Reply:
Same as above, but an example where -L is not specified. Does the forward iterator start from the first row if -U is not specified?